src/pages/Validators/Index.tsx

```diff
@@ -30,23 +30,37 @@ export default function ValidatorsPage() {
         <>
           <ValidatorsMap />
           {inDev && (
-            <Tabs value={onDelegatory} onChange={handleTabChange}>
-              <Tab label="All Nodes" value={false} />
-              <Tab label="Delegation Nodes" value={true} />
-            </Tabs>
+            <ToggleButtonGroup
+              sx={{width: 1 / 3, height: 50}}
```

Comment:
How does width: 1/3 look on mobile?

Reply:
good call, on mobile, text gets to be two lines. what would be a mobile friendly way to set the size and width on these buttons?

Reply:
Personally I would use flexbox and a media query but it looks like the way to do it in emotion is https://theme-ui.com/sx-prop#responsive-values.